packages/r/rabbitmq-c/xmake.lua

```diff
@@ -0,0 +1,41 @@
+package("rabbitmq-c")
+
+    set_homepage("https://github.com/alanxz/rabbitmq-c")
+    set_description("This is a C-language AMQP client library for use with v2.0+ of the RabbitMQ broker.")
+
+    set_urls("https://github.com/alanxz/rabbitmq-c/archive/refs/tags/$(version).zip",
+             "https://github.com/alanxz/rabbitmq-c.git")
+
+    add_versions("v0.11.0", "fad876075cbcf9a2f3dff27be66391f69117d5068529ba1d53e67fe59c2d88e1")
+
+    if is_plat("windows") then
+        add_configs("shared", {description = "Build shared library.", default = false, type = "boolean"})
+    elseif is_plat("macosx", "linux") then
+        add_deps("autoconf", "automake", "pkg-config")
+    end
+    add_configs("tools", {description = "Build the command line tools.", default = false, type = "boolean"})
+    add_configs("ssl", {description = "Build rabbitmq-c with SSL support.", default = false, type = "boolean"})
```

Comment:
you need add `package:add("deps", "openssl")` if `package:config("ssl")` is enabled in on_load()